wallet-new/src/Cardano/Wallet/API/V1/Migration/Types.hs

```diff
@@ -224,9 +238,23 @@ instance Migrate V0.CTx V1.Transaction where
 
         let V0.CTxMeta{..} = ctMeta
         txCreationTime <- eitherMigrate ctmDate
+        txStatus <- eitherMigrate ctCondition
 
         pure V1.Transaction{..}
 
+instance Migrate V0.CPtxCondition V1.TransactionStatus where
+    eitherMigrate = Right . \case
+        V0.CPtxCreating{} ->
+            V1.Creating
+        V0.CPtxApplying{} ->
+            V1.Applying
+        V0.CPtxInBlocks{} ->
+            V1.InNewestBlocks
+        V0.CPtxWontApply{} ->
+            V1.WontApply
+        V0.CPtxNotTracked ->
+            V1.Persisted
```

Comment:
Cannot really suggest a meaningful mapping for this one (not as any arbitrary than `V1.Persisted` :trollface: ). The documentation for `CPtxNotTracked` is value and I can see it's not even actively used AFAICS, so this is fine I think.

Reply:
Yeah :man_shrugging: I think the important thing is to pick a nice representation for the new data layer and force the old one to fit, rather than try and force the new one to conform to the old.